widget/index.html

```diff
@@ -571,8 +571,8 @@
 
         if (bgImage) {
             let url = buildfire.imageLib.cropImage(bgImage, {
-                    width: window.innerWidth,
-                    height: document.documentElement.clientHeight
+                    width: buildfire.getContext().initialWidth || window.innerWidth,
+                    height: buildfire.getContext().initialHeight || document.documentElement.clientHeight

```

Comment:
let's use window.innerHeight as fallback.

Reply:
window.innerHeight is reported to cause problems with cordova / iOs devices, document.documentElement.clientHeight  will return the same values, but shouldn't cause any problem

Reply:
sounds good.